angular/core/components/thread_page/activity_card/discussion_moved.haml

```diff
@@ -0,0 +1,10 @@
+.thread-item{ng-controller: "DiscussionMovedItemController"}
+  .thread-item__header.media
```

Comment:
oh wow. ok. Thanks for that. Let me just say that I'd like to move away from bootstrap stuff (we don't use much now). This is fine if all the rest use the same format (should be pretty easy to remove/replace later), but new code should avoid bootstrap unless it provides incredible value.